### PR TITLE
fix: mistakes in spell checking action

### PR DESCRIPTION
**Pull request by LostNeophyte**
there were missing words in allow.txt, my branch was behind

### CODE REVIEW

**Comment by LostNeophyte** [comment]
@kylo252 our deploy action has the same warning btw

File: .github/workflows/spelling.yml [hunk 1]
```diff
@@ -15,9 +15,6 @@ on:
     - 'opened'
     - 'reopened'
     - 'synchronize'
-  issue_comment:
```
Comment (kylo252) [inline .github/workflows/spelling.yml:18]: we missed line 74, it should also be changed  😄 

```diff
diff --git a/.github/workflows/spelling.yml b/.github/workflows/spelling.yml
index 2696e09..d594e20 100644
--- a/.github/workflows/spelling.yml
+++ b/.github/workflows/spelling.yml
@@ -74,7 +74,7 @@ jobs:
     if: (success() || failure()) && needs.spelling.outputs.followup && contains(github.event_name, 'pull_request')
     steps:
     - name: comment
-      uses: check-spelling/check-spelling@v0.0.21-alpha1
+      uses: check-spelling/check-spelling@prerelease
       with:
         checkout: true
         task: ${{ needs.spelling.outputs.followup }}
```

Comment (LostNeophyte) [reply .github/workflows/spelling.yml:18]: it should be `v0.0.21-alpha1`, the maintainer created it just for us

Comment (LostNeophyte) [reply .github/workflows/spelling.yml:18]: done

**Review by kylo252 — APPROVED** [review]
Thanks!

**Pull request merged**